HealthMonitoring.Management.Core.UnitTests/EndpointRegistryTests.cs

```diff
@@ -13,13 +13,13 @@ public class EndpointRegistryTests
         private readonly EndpointRegistry _registry;
         private readonly Mock<IHealthMonitorTypeRegistry> _healthMonitorTypeRegistry;
         private readonly Mock<IEndpointConfigurationRepository> _configurationStore;
-        private readonly Mock<IEndpointStatsRepository> _statsRepository;
+        private readonly Mock<IEndpointStatsPersistCoordinator> _statsRepository;

```

Comment:
inconsistent field to class name (this applies to the changed code below as well)